en/advanced_config/pid_tuning_guide_multicopter.md

```diff
@@ -83,143 +215,24 @@ might not climb or lose altitude a bit, but it will never flip over. The
-(ATTITUDE.rollspeed/pitchspeed) from **Widgets -> Analyze**. It must not oscillate, but some
-overshoot (10-20%) is ok.
-
-
-#### I Gain Tuning
-
-If the roll and pitch rates never reach the setpoint but have an offset,
-add [MC_ROLLRATE_I](../advanced_config/parameter_reference.md#MC_ROLLRATE_I) and [MC_PITCHRATE_I](../advanced_config/parameter_reference.md#MC_PITCHRATE_I) gains, starting at 5-10% of the `MC_ROLLRATE_P` gain value.
-
-
-### Step 3: Stabilize Roll and Pitch Angles
-
-#### P Gain Tuning
-
-Parameters: [MC_ROLL_P](../advanced_config/parameter_reference.md#MC_ROLL_P), [MC_PITCH_P](../advanced_config/parameter_reference.md#MC_PITCH_P).
-
-- Set `MC_ROLL_P` and `MC_PITCH_P` to a small value, e.g. 3
-
-Keep the multi rotor in your hand and increase the thrust to about 50%,
-so that the weight is virtually zero. Tilt it in roll or pitch
-direction, and observe the response. It should go slowly back to level.
-If it oscillates, tune down P. Once the control response is slow but
-correct, increase P until it starts to oscillate. Optimal response is
-some overshoot (\~10-20%). After getting stable response fine tune
-`RATE_P`, `RATE_D` again.
-
-In *QGroundControl* you can plot roll and pitch (`ATTITUDE.roll`/`ATTITUDE.pitch`) and
-control (ctrl0, ctrl1). Attitude angles overshoot should be not more than
-10-20%.
-
-
-### Step 4: Stabilize Yaw Rate
-
-#### P Gain Tuning
-
-Parameters: [MC_YAWRATE_P](../advanced_config/parameter_reference.md#MC_YAWRATE_P).
-
-- Set `MC_YAWRATE_P` to small value, e.g. 0.1
-
-Keep the multi rotor in your hand and increase the thrust to about 50%,
-so that the weight is virtually zero. Turn it around its yaw axis,
-observe the response. The motor sound should change and the system
-should fight the yaw rotation. The response will be substantially weaker
-than roll and pitch, which is fine. If it oscillates or becomes twitchy,
-tune down `RATE_P`. If response is very large even on small movements
-(full throttle spinning vs idle spinning propellers) reduce `RATE_P`.
-Typical value is around 0.2…0.3.
-
-The yaw rate control, if very strong or oscillating, can deteriorate the
-roll and pitch response. Check the total response by turning around,
-roll, pitch and yaw.
-
-
-### Step 5: Stabilize Yaw Angle
-
-
-#### P Gain Tuning
-
-Parameters: [MC_YAW_P](../advanced_config/parameter_reference.md#MC_YAW_P).
-
-- Set `MC_YAW_P` to a low value, e.g. 1
-
-Keep the multi rotor in your hand and increase the thrust to about 50%,
-so that the weight is virtually zero. Rotate it around yaw, and observe
-the response. It should go slowly back to the initial heading. If it
-oscillates, tune down P. Once the control response is slow but correct,
-increase P until the response is firm, but it does not oscillate.
-Typical value is around 2…3.
-
-Look at `ATTITUDE.yaw` in *QGroundControl*. Yaw overshoot should be not more
-than 2-5% (which is less than the overshoot for roll and pitch angles).
 
+<!-- TODO

```

Comment:
@Stifael, @MaEtUgR, @RomanBapst can one of you write the Position controller tuning part?